### PR TITLE
streamlit-condition-tree 0.1.3

### PR DESCRIPTION
**Pull request by psteyer**
steamlit-condition-tree 0.1.3 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4475](https://anaconda.atlassian.net/browse/PKG-4475) 
- [Upstream repository](https://github.com/cedricvlt/streamlit-condition-tree)
- [pypi](https://pypi.org/project/streamlit-condition-tree/)

### Explanation of changes:

- `pypi` uses a `.whl` file instead of a `tar` - So I build the recipe from scratch due to `conda skeleton` not being able to find anything.

[PKG-4475]: https://anaconda.atlassian.net/browse/PKG-4475?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/cedricvlt/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5895a9c0546cf5facc9db0f84d76310403c22da291c99ec0519235d12c5e480b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [py<38 or s390x]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<37 or s390x]
```
https://github.com/cedricvlt/streamlit-condition-tree/blob/e1eccc29f407fbd768405d26ebfedf59a476631e/setup.py#L20

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by Jrice1317 — APPROVED** [review]
Outside of Serhii's comment, LGTM!

**Review by ViridianMelody — APPROVED** [review]
LGTM!

**Pull request merged**